README.md

```diff
@@ -44,5 +52,18 @@ To run all checks execute the following command:
 ./gradlew check
 ```
 
+## Job management
+
+The service manages its clients' jobs with (Quartz)[http://www.quartz-scheduler.org/].
+It uses a PostgreSQL database for persisting those jobs, so that they will not be lost
+if the server goes down. Also, Quartz is configured to run in cluster mode, i.e. the
```

Comment:
I'd remove the 'so that they will not be lost if the server goes down.' bit

Reply:
Done